src/components/common/MainMenu.js

```diff
@@ -41,7 +41,7 @@ const MainMenu = () => {
   }
 
   if (isLoading || rootItemIsLoading) {
-    return <Loader />;
+    return <Skeleton variant="text" />;
   }
```

Comment:
Should we include the skeleton in `DynamicTreeView` instead? We will probably need a new param `isLoading`.